crates/noirc_evaluator/src/ssa/code_gen.rs

```diff
@@ -545,6 +566,27 @@ impl IRGenerator {
                 }
                 Ok(Value::Single(new_var))
             }
+            Expression::Literal(Literal::Str(string)) => {
+                let string_as_integers = string
+                    .bytes()
+                    .map(|byte| {
+                        let f = FieldElement::from_be_bytes_reduce(&[byte]);
+                        Expression::Literal(Literal::Integer(
+                            f,
+                            Type::Integer(noirc_frontend::Signedness::Unsigned, 8),
+                        ))
+                    })
+                    .collect::<Vec<_>>();
```

Comment:
`.map(...).collect::<Vec<_>>()` can be a `vecmap`

Reply:
Done